Docs/Verbali Interni/VI_2024-12-13/content.tex

```diff
@@ -0,0 +1,50 @@
+% Insert content here
+\section{Informazioni generali}
+\subsection{Dettagli sull'incontro}
+\begin{itemize}
+    \item \textbf{Luogo}: Discord
+    \item \textbf{Data}: 13-12-2024
+    \item \textbf{Ora di inizio}: 16:30
+    \item \textbf{Ora di fine}: 17:15
+    \item \textbf{Partecipanti}:
+    \begin{itemize}
+        \item Bergamin Elia
+        \item Chilese Elena
+        \item Diviesti Filippo
+        \item Djossa Edgar
+        \item Pincin Matteo 
+        \item Soranzo Andrea  
+    \end{itemize}
+\end{itemize}
+
+\section{Motivo della riunione}
+Questo incontro è servito per confrontarci sul lavoro svolto a metà del terzo sprint, riportare eventuali dubbi da parte dei membri del team 
+e discuterli per arrivare a una soluzione.\\
+In particolare, abbiamo deciso come riportare le metriche nelle \textit{Norme di progetto}, abbiamo discusso sui diagrammi UML relativi ai casi d'uso, come gestire l'automazione del glossario e come ristrutturare il repository dopo la RTB.
+\\Inoltre, è stato discusso come organizzare il prossimo sprint, considerando che si svolgerà durante il periodo delle festività natalizie, al fine di garantire una pianificazione efficace e il rispetto delle scadenze.
+
+\section{Resoconto}
+% insert here all the steps
+\subsection{Metriche}
+Ci siamo chiesti se la sezione delle metriche dovesse essere riportata come sezione autonoma o come sottosezione all'interno delle \textit{Norme di progettto}, abbiamo deciso che inseriremo una sezione dedicata alle metriche.
+
+\subsection{Sprint 4}
+Dopo aver evidenziato che lo sprint successivo si svolgerà nel periodo che va dal 23 dicembre 2024 al 3 gennaio 2025, e quindi in concomitanza con le festività natalizie, ci siamo confrontati per discutere le disponibilità di ciascun membro del team. Ogni partecipante ha espresso la propria disponibilità a lavorare al progetto durante lo sprint, e abbiamo deciso che i ruoli assegnati e le rispettive ore di lavoro saranno definiti in base alle disponibilità individuali, al fine di avviare il lavoro sul PoC il prima possibile.\\
+Abbiamo discusso anche i criteri per l'assegnazione dei ruoli nel prossimo sprint. In particolare, il ruolo di programmatore sarà affidato ai membri che hanno acquisito maggiore dimestichezza con le tecnologie richieste dal progetto, ovvero React e Three.js, e che daranno la propria disponibilità a lavorare nel periodo dal 23 dicembre 2024 al 3 gennaio 2025.  
```

Comment:
Direi in modo più conciso:  

Poiché il prossimo sprint si terrà dal 23 dicembre 2024 al 3 gennaio 2025, in concomitanza con le festività natalizie, abbiamo discusso le disponibilità di ciascun membro del team per gestire al meglio l'assegnazione dei ruoli e delle ore di lavoro. In particolare, il ruolo di programmatore sarà affidato ai membri che hanno acquisito maggiore dimestichezza con le tecnologie React e Three.js e che daranno disponibilità nel periodo indicato, così da avviare il PoC al più presto.